theme/src/components/header.js

```diff
@@ -45,7 +45,7 @@ function Header({location, isSearchEnabled = true}) {
         bg="#333333"
       >
         <Flex alignItems="center">
-          <Link as={GatsbyLink} to="/" style={logoStyle} mr={3}>
+          <Link as={GatsbyLink} to="/" style={logoStyle} mr={3} title='NPM home'>
```

Comment:
Please change the title to `npm Docs`

Also, npm should always be lowercase and not NPM :  https://github.com/npm/cli#faq-on-branding

Reply:
Please see https://github.com/npm/documentation/pull/253#issuecomment-1282300049